### PR TITLE
Countdown timer

### LINKED ISSUE #17: Implement the countdown timer

### PR DESCRIPTION
**Pull request by Mester**
fixes #17

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,4 @@
 Django==1.8.4
+python-dateutil==2.4.2
 requests==2.7.0
+six==1.9.0
```
Comment (adrianmoisey) [inline requirements.txt]: Why do we need six?

File: tvshowcountdown/tvshow/templates/index.html [hunk 2]
```diff
@@ -21,7 +21,17 @@
 {% endif %}
 
 {% if countdown_timer %}
-    <p> {{ countdown_timer }} </p>
+    <script src=/static/js/countdown.js></script>
```
Comment (adrianmoisey) [inline tvshowcountdown/tvshow/templates/index.html]: I think there is a static url variable that we can use, won't be useful to us though.

**Comment by adrianmoisey** [comment]
:+1: :+1:

File: requirements.txt (hunk 1 above)
Comment (Mester) [inline requirements.txt]: python-dateutil was dependent on it I believe

File: tvshowcountdown/tvshow/templates/index.html (hunk 2 above)
Comment (Mester) [inline tvshowcountdown/tvshow/templates/index.html]: Yeah I think so, tried to get it working but couldn't manage

**Pull request merged**